demo/app.js

```diff
@@ -7,13 +7,17 @@ import CKEditorInspector from '@ckeditor/ckeditor5-inspector';
 
 import Math from "../src/math";
 
-ClassicEditor.create(document.querySelector("#editor"), {
-	plugins: [Essentials, Paragraph, Bold, Italic, Math],
-	toolbar: ["bold", "italic", "math"],
-	math: { engine: "katex", mathLiveSettings: { enabled: true } },
-})
-	.then((editor) => {
-		console.log("Editor was initialized", editor);
+ClassicEditor.create( document.querySelector( '#editor' ), {
+	plugins: [ Essentials, Paragraph, Bold, Italic, Math ],
+	toolbar: [ 'bold', 'italic', 'math' ],
+	math: {
+		engine: 'katex',
+		mathLiveSettings: { enabled: true },
+		textareaInput: true
```

Comment:
IMO this shouldn't be configurable. Latex is usually long. Using input wasn't appropriate in the first place.

Reply:
https://github.com/bethinkpl/ckeditor5-math/pull/6/commits/267897c6fe90c4177974871a0829b238d26a94e5